src/_pytest/_code/code.py

```diff
@@ -676,7 +679,7 @@ def repr_locals(self, locals):
                     if self.truncate_locals:
                         str_repr = self._saferepr(value)
                     else:
-                        str_repr = pprint.pformat(value)
+                        str_repr = self._safeformat(value)
```

Comment:
Why not use it directly?
I.e. not via `_safeformat` method - I see that this is done for `_saferepr`, but I do not think it is good to follow.

Reply:
Yeah, I've just followed the way it has been called. I will change this to a direct call of `safeformat` and `saferepr`. Thank you.